examples/ijrr2014_rec_hor/planner.py

```diff
@@ -142,7 +145,7 @@ def construct_local_ts(self):
 		ts.init[center_cell] = 1
 
 		# Define the vertices of the local TS (the sensing cells)
-		for cx, cy in it.product(range(0, self.quad.sensing_range), repeat=2):
+		for cx, cy in it.product(list(range(0, self.quad.sensing_range)), repeat=2):
```

Comment:
The `list` transformation is not needed here.  Restore code to previous version.